packages/build/package-lock.json

```diff
@@ -38,32 +38,35 @@
       }
     },
     "node_modules/@ampproject/remapping": {
-      "version": "2.2.0",
-      "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.2.0.tgz",
-      "integrity": "sha512-qRmjj8nj9qmLTQXXmaR1cck3UXSRMPrbsLJAasZpF+t3riI71BXed5ebIOYwQntykeZuhjsdweEc9BxH5Jc26w==",
+      "version": "2.2.1",
```

Comment:
If you're planning on rolling these versions forward then this build should be put on all of our builds that use `@labkey/build` to ensure they work with these updates. This is akin to a package update just that we're not updating any of the direct dependencies. 

If you don't intend to do this then I'd say just roll back these version changes within the `package-lock.json` and we'll pick them up the next time we do package updates.

Reply:
This was backed out